languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -10069,6 +10069,34 @@ though, and has since been slightly modified)-->
     </rulegroup>
   </category>
   <category id="CAT_GRAMMAIRE" name="Grammaire">
+      <rulegroup id="QUELLES_OU_QU_ELLES" name="quelles ou qu'elles" default="temp_off">
+      <rule>
+          <pattern>
+              <token postag="SENT_START"/>
+              <marker>
+                  <token regexp="yes">qu'?</token>
+                  <token>elles</token>
+              </marker>
+              <token postag="V.*" postag_regexp="yes" skip="-1"><exception postag="V.* sub*|V avoir sub.*|V etre sub.*" postag_regexp="yes"></exception></token>
```

Comment:
The "string" of the exception is "empty". This means, instead of

`<exception postag="V.* sub*|V avoir sub.*|V etre sub.*" postag_regexp="yes"></exception>`

you need to write

`<exception postag="V.* sub*|V avoir sub.*|V etre sub.*" postag_regexp="yes"/>`.

Your postag within the exception contains a typo:

you wrote `sub*` instead of `sub.*`. Also, if you have `V.*sub.*`, this will already match `V avoir sub.` and `V etre sub.`, because of the `.*` between `V` and `sub`.